src/MongoDB/ReadPreference.c

```diff
@@ -147,7 +147,7 @@ zend_object_value php_phongo_readpreference_create_object(zend_class_entry *clas
 HashTable *php_phongo_readpreference_get_debug_info(zval *object, int *is_temp TSRMLS_DC) /* {{{ */
 {
 	zval                 retval = zval_used_for_init;
-	mongoc_read_prefs_t *read_prefs = phongo_read_preference_from_zval(object TSRMLS_CC);
+	const mongoc_read_prefs_t *read_prefs = phongo_read_preference_from_zval(object TSRMLS_CC);
```

Comment:
I originally thought of getting rid of this declaration and just passing the return value directly below. I then realized that `php_phongo_read_preference_to_zval()` doesn't actually check that this is a valid pointer, and `phongo_read_preference_from_zval()` may return `NULL`. Should we add a sanity check here?